joern-cli/src/main/scala/io/joern/joerncli/JoernVectors.scala

```diff
@@ -121,14 +121,14 @@ object JoernVectors extends App {
     Using.resource(CpgBasedTool.loadFromOdb(config.cpgFileName)) { cpg =>
       val embedding = new BagOfAPISymbolsForMethods().embed(cpg)
       println("{")
-      println("objects:")
+      println("\"objects\":")
       traversalToJson(embedding.objects)
       if (config.dimToFeature) {
-        println("dimToFeature: ")
+        println(",\"dimToFeature\": ")
         println(Serialization.write(embedding.dimToStructure))
         println(",")
       }
-      println("vectors:")
+      println(",\"vectors\":")
```

Comment:
With config.dimToFeature we would end up with two consecutive ','.

Reply:
thanks!